File: angular-spring-boot/backend/src/main/java/ru/isnagornov/templates/rest/EntityCrudRestController.java
package ru.isnagornov.templates.rest;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import ru.isnagornov.templates.entity.Entity;
import ru.isnagornov.templates.mapper.EntityMapper;
import ru.isnagornov.templates.service.EntityService;

@RestController
@RequestMapping(path = "entity")
public class EntityCrudRestController extends AbstractController<Entity, EntityMapper, EntityService> {

    public EntityCrudRestController(EntityService service) {
        super(service);
    }

}

File: angular-spring-boot-kotlin/backend/src/main/kotlin/ru/isnagornov/templates/rest/EntityCrudRestController.kt
package ru.isnagornov.templates.rest

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import ru.isnagornov.templates.entity.Entity
import ru.isnagornov.templates.mapper.EntityMapper
import ru.isnagornov.templates.service.EntityService

@RestController
@RequestMapping(path = ["entity"])
class EntityCrudRestController(service: EntityService) : AbstractController<Entity, EntityMapper, EntityService>(service)
